.github/workflows/python-package.yml

```diff
@@ -7,31 +7,32 @@ on: [push]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
       matrix:
         python-version: ["3.10"]
 
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v3
-      with:
-        python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install flake8 pytest
-        pip install .
-        pip install ./[hyper]
-    - name: Check linting with flake8
-      run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
-    - name: Run pytest on tableau-utilities
-      run: |
-        cd tableau_utilities && pytest -v
+      - uses: actions/checkout@v3
```

Comment:
These changes are mostly linter changes as I ran my yaml linter.  I can show you that if you want to start linting yaml here